examples/js/effects/VREffect.js

```diff
@@ -102,16 +102,26 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	document.addEventListener( fullscreenchange, function () {
 
-		isPresenting = isDeprecatedAPI && vrHMD && ( document.mozFullScreenElement || document.webkitFullscreenElement ) !== undefined;
+		isPresenting = vrHMD && ( vrHMD.isPresenting || isDeprecatedAPI && ( document.mozFullScreenElement || document.webkitFullscreenElement ) !== undefined );
```

Comment:
We should add some parenthesis to make this code more clear...

Are we doing `X && ( ( Y || Z ) && W )` or `X && ( Y || ( Z && W ) )`?